### PR TITLE
my second commit

### PR DESCRIPTION
**Pull request by Clementify**
I am done with my Lesson 1 and 2 task2 please come and review @ChidiPatrick and @Promise-J

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="../css/style.css">
+        <title>Document</title>
+    </head>
+
+    <body>
+        <header>
+            <h1>OKORO GLOBAL BUSINESS</h1>
+            <img src="/tailoring material.jpg" alt="tailoring material" class="img">
```
Comment (ChidiPatrick) [inline html/index.html]: Close your  image tag

File: html/index.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                <a href="#">HOME</a>
+                <a href="#">ABOUT</a>
+                <a href="#">CONTACT US</a>
+            </nav>
+        </header>
+        <aside>
+            <section>
+                <article>
+                    <h2>ALL YOU NEED TO KNOW ABOUT TAILORING MATERIAL BUSINESS</h2>
+
```
Comment (ChidiPatrick) [inline html/index.html]: Please indent your code proper. The space is too much

File: html/index.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                        To feel and identify it, suppose that any place where the threads and needles are used means that they are somehow connected to the stitching industry. If so, then understand there will be huge potential to start a business activity with them.
+                        
+                        Further, Suppose you are already a part of this particular tailoring business, online sewing business, and sewing jobs. In that case, you will need some sewing accessories or sewing tools, through which the product is given a complete and final shape.
+                        
+                        Let us give you an example of a particular design clothes style, in which various materials and accessories are needed to provide them with a unique figure. In that case, you have to contact the merchant who supplies the sewing material; this is how its importance reflects. 
+                        
+
```
Comment (ChidiPatrick) [inline html/index.html]: space is too much

**Review by ChidiPatrick — CHANGES_REQUESTED** [review]
Please work on your indentation, and place your contents in proper elements

File: html/index.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+            <nav>
+                <a href="#">HOME</a>
+                <a href="#">ABOUT</a>
+                <a href="#">CONTACT US</a>
+            </nav>
+        </header>
+        <aside>
+            <section>
+                <article>
+                    <h2>ALL YOU NEED TO KNOW ABOUT TAILORING MATERIAL BUSINESS</h2>
```
Comment (ChidiPatrick) [inline html/index.html:27]: Work on your indentation

**Review by ChidiPatrick — CHANGES_REQUESTED** [review]
Work on your indentation

File: html/index.html (hunk 4 above)
Comment (Clementify) [reply html/index.html:27]: Alright sir.

**Review by ChidiPatrick — APPROVED** [review]
nice work

**Pull request merged**